.github/workflows/tracks.yml

```diff
@@ -21,6 +21,60 @@ jobs:
             flags: --test-mode --challenge=append-no-conflicts
             stack-version: "8.3.2"
 
+          - name: elastic/security (index-alert-source-events)
+            track: elastic/security
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=index-alert-source-events
+            stack-version: "8.3.2"
+          - name: elastic/security (security-indexing-querying)
+            track: elastic/security
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=security-indexing-querying
+            stack-version: "8.3.2"
+          - name: elastic/security (security-indexing)
+            track: elastic/security
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=security-indexing
+            stack-version: "8.3.2"
+          - name: elastic/security (security-querying)
+            track: elastic/security
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=security-querying
+            stack-version: "8.3.2"
+
+          - name: elastic/logs (logging-disk-usage)
+            track: elastic/logs
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=logging-disk-usage
+            stack-version: "8.3.2"
+          - name: elastic/logs (logging-indexing-querying)
+            track: elastic/logs
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=logging-indexing-querying
+            stack-version: "8.3.2"
+          - name: elastic/logs (logging-indexing)
+            track: elastic/logs
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=logging-indexing
+            stack-version: "8.3.2"
+          - name: elastic/logs (logging-querying)
+            track: elastic/logs
+            flags: --test-mode --track-params="number_of_replicas:0" --challenge=logging-querying
+            stack-version: "8.3.2"
+          - name: elastic/logs (logging-snapshot-restore)
```

Comment:
can we drop this one? I'll implement it soon as a part of #282

Reply:
Done